### ISSUE TITLE
Start Galxe campaings to find ACTIVE/REAL users

### ISSUE THREAD

**Issue by sia013** [scroll-tech/contribute-to-scroll#593]
### Name of the project

Scroll Interactions

### Project type

Library

### Site of the project

scroll.io

### Github repository of the project

https://github.com/scroll-tech/contribute-to-scroll

### Networks

Scroll Mainnet

### Additional context

galxe is very good space to find Real/human/active users for future Ardrops
in my opinion , we missed scroll there ;)
linea and taiko are the good examples for this suggestion
many people can find new dapps on scroll to use them and it will be perfect to involve many users in mainnet
i think this will be the smart move to get more tvl for netwok too

**Comment by honglek06** [comment]
No, pls no galxe. the platform is so buggy and frustrating from time to time

**Comment by sia013** [comment]
> No, pls no galxe. the platform is so buggy and frustrating from time to time

Agree with you that galxe is buggy, but still have many users , it's just a suggestion, layer 3 , interact are still good options , but at the end campaigns can help user to find out more about scroll
